introduction.md

```diff
@@ -1,2 +1,8 @@
 Hi. This is Cari. My favorite codeschool path is PHP or Git Real. 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?